docs/customize/toc.md

```diff
@@ -154,6 +154,17 @@ sections:
 
 The primary difference is that the `jb-book` format uses `parts:` and `chapters:` syntax, while the `jb-article` format uses `sections:` syntax alone.
 
+To build an article (or a book) that is comprised of multiple files
+so that it is rendered as a single HTML document containing all the content,
+you can use of the
+[singlehtml builder](https://www.sphinx-doc.org/en/master/usage/builders/index.html#sphinx.builders.singlehtml.SingleFileHTMLBuilder).
```

Comment:
```suggestion
To build a single HTML page from these files (rather than one page per file), use the [singlehtml builder](https://www.sphinx-doc.org/en/master/usage/builders/index.html#sphinx.builders.singlehtml.SingleFileHTMLBuilder).
```